packages/frontend/src/components/CompilationHistory/CompilationHistoryTable.tsx

```diff
@@ -384,25 +385,6 @@ const CompilationHistoryTable: FC<CompilationHistoryTableProps> = ({
             sorting,
         },
         onSortingChange: setSorting,
-        renderBottomToolbar: () => (
-            <Box>
-                <Group gap="xs" px="md" py="xs">
-                    {isFetching ? (
-                        <>
-                            <Loader size={14} c="gray" />{' '}
-                            <Text size="xs" c="dimmed">
-                                Loading...
-                            </Text>
-                        </>
-                    ) : (
-                        <Text size="xs" c="dimmed">
-                            Showing {totalFetched} of {totalDBRowCount} log
-                            {totalDBRowCount === 1 ? '' : 's'}
-                        </Text>
-                    )}
-                </Group>
-            </Box>
-        ),
```

Comment:
this + no useEffect were the culprits. 
This should just work but I fear that state setting (internally by the library) isn't properly configured for the React ecosystem. The removal of this isn't too concerning but the usage of this library is. Thinking about alternatives or an in-house solution with already installed libraries: mantine + tanstack table